__tests__/integration/previewRDFHelper.js

```diff
@@ -27,3 +27,8 @@ export async function fillInRequredFieldsForBibframeInstance() {
   await page.waitForSelector('#rbt-menu-item-0')
   await page.click('#rbt-menu-item-0')
 }
+
+export async function incompleteFieldsForBibframeInstance() {
+  // This assertion adds 1 to each it blocks assertion count
```

Comment:
PR looks fine but I don't understand this comment.

Reply:
I'm not great and clarifying this, but this is the assertion that means any block "it" block that includes this call needs to check for it's assertions+1.

Reply:
ah!  how about

```
// this assertion adds 1 to each "it" block's expect.assertion(n) count
```